src/dotty/tools/dotc/parsing/Parsers.scala

```diff
@@ -4,7 +4,7 @@ package parsing
 
 import scala.collection.mutable.ListBuffer
 import scala.collection.immutable.BitSet
-import util.{ SourceFile, SourcePosition }
+import util.{SourceFile, SourcePosition}
```

Comment:
Should have been squished into 1282d77f41976355dc00dd87d62f2c28c3e5b90f, but whatever - it's not the end of the world.